cba_settings.sqf

```diff
@@ -156,55 +179,52 @@ force force ace_maptools_rotateModifierKey = 1;
-force force ace_medical_allowUnconsciousAnimationOnTreatment = false;
-force force ace_medical_amountOfReviveLives = -1;
 force force ace_medical_bleedingCoefficient = 1;
-force force ace_medical_blood_enabledFor = 2;
-force force ace_medical_consumeItem_PAK = 1;
-force force ace_medical_consumeItem_SurgicalKit = 0;
-force force ace_medical_delayUnconCaptive = 3;
-force force ace_medical_enableAdvancedWounds = true;
-force force ace_medical_enableFor = 1;
-force force ace_medical_enableOverdosing = true;
-force force ace_medical_enableRevive = 0;
-force force ace_medical_enableScreams = false;
-force force ace_medical_enableUnconsciousnessAI = 0;
-force force ace_medical_enableVehicleCrashes = true;
-force force ace_medical_healHitPointAfterAdvBandage = false;
-force force ace_medical_increaseTrainingInLocations = true;
-force force ace_medical_keepLocalSettingsSynced = true;
-force force ace_medical_level = 2;
-force force ace_medical_litterCleanUpDelay = 0;
-force force ace_medical_litterSimulationDetail = 3;
-force force ace_medical_maxReviveTime = 120;
-force force ace_medical_medicSetting = 2;
-force force ace_medical_medicSetting_basicEpi = 1;
-force force ace_medical_medicSetting_PAK = 1;
-force force ace_medical_medicSetting_SurgicalKit = 1;
-force force ace_medical_menu_allow = 1;
-force force ace_medical_menu_maxRange = 3;
-force force ace_medical_menu_openAfterTreatment = true;
-//ace_medical_menu_useMenu = 0;
-//ace_medical_menuTypeStyle = 0;
-force force ace_medical_moveUnitsFromGroupOnUnconscious = false;
+force ace_medical_blood_bloodLifetime = 900;
+force ace_medical_blood_enabledFor = 2;
+force ace_medical_blood_maxBloodObjects = 500;
+force force ace_medical_fatalDamageSource = 0;
+force force ace_medical_feedback_painEffectType = 0;
+force force ace_medical_fractures = 1;
+//ace_medical_gui_enableActions = 0;
+//ace_medical_gui_enableMedicalMenu = 1;
+//ace_medical_gui_enableSelfActions = true;
+force force ace_medical_gui_maxDistance = 3;
+//ace_medical_gui_openAfterTreatment = true;
+force force ace_medical_ivFlowRate = 1;
+force force ace_medical_limping = 1;
 force force ace_medical_painCoefficient = 1;
-//force force ace_medical_painEffectType = 0;
-force force ace_medical_painIsOnlySuppressed = true;
 force force ace_medical_playerDamageThreshold = 1;
-force force ace_medical_preventInstaDeath = true;
-force force ace_medical_remoteControlledAI = true;
-force force ace_medical_useCondition_PAK = 1;
-force force ace_medical_useCondition_SurgicalKit = 0;
-force force ace_medical_useLocation_basicEpi = 0;
-force force ace_medical_useLocation_PAK = 0;
-force force ace_medical_useLocation_SurgicalKit = 0;
-
-// ACE Mk6 Mortar
-force force ace_mk6mortar_airResistanceEnabled = false;
-force force ace_mk6mortar_allowCompass = true;
-force force ace_mk6mortar_allowComputerRangefinder = false;
-force force ace_mk6mortar_useAmmoHandling = false;
+force force ace_medical_spontaneousWakeUpChance = 0.05;
```

Comment:
Suggest this is increased to 0.15, which gives you a 47% of getting up after 1 minutes, 85% of getting up after 3 minutes, 96% of getting up after 5 minutes. Keep in mind people still need to get to a "stable" state first, which may take some time as well